app/src/main/java/blockeq/com/stellarwallet/activities/SendActivity.kt

```diff
@@ -87,32 +116,22 @@ class SendActivity : BasePopupActivity(), NumberKeyboardListener {
         }
     }
 
-    /**
-     * Shows amount in UI.
-     */
     private fun showAmount(amount: String) {
-        amount_text.text = if (amount.isEmpty()) "0" else amount
+        amountTextView.text = if (amount.isEmpty()) "0" else amount
     }
 
-    /**
-     * Calculate the number of decimals of the string.
-     */
-    private fun getNumDecimals(num: String): Int {
-        return if (!hasDecimalPoint(num)) {
-            0
-        } else num.substring(num.indexOf('.') + 1, num.length).length
-    }
+    //endregion
 
-    /**
-     * Checks whether the string has a comma.
-     */
-    private fun hasDecimalPoint(text: String): Boolean {
-        for (i in 0 until text.length) {
-            if (text[i] == '.') {
-                return true
-            }
-        }
-        return false
+    //region Horizon callbacks
+    override fun OnSendSuccess() {
```

Comment:
Is there a provision for refreshing the transaction history after a successful send? Or we are depending on the polling for a refresh? As a reminder, let's put a TODO for polling for only non-created accounts on Stellar. For accounts created on Stellar, we'll need to implement streaming - let's chat about this one.

Reply:
Right now it's depending on polling for refreshes so we can talk about it tomorrow?

Reply:
@Daniel-Wang definitely. Doesn't have to be this PR. I've added this in AirTable so we can get it in the next spring planning.